pyresample/bucket/__init__.py

```diff
@@ -234,7 +234,8 @@ def get_average(self, data, fill_value=np.nan, mask_all_nan=False):
         LOG.info("Get average value for each location")
 
         sums = self.get_sum(data, mask_all_nan=mask_all_nan)
-        counts = self.get_count()
+        counts = self.get_sum(np.logical_not(np.isnan(data)).astype(int),
+                              mask_all_nan=False)
 
         average = sums / counts
```

Comment:
Shall we mask zero counts before division in order to avoid dask warnings?

Reply:
Yes, that could make the output cleaner for the user.

Reply:
Done